models/junos/l3_interfaces/junos_l3_interfaces.yml

```diff
@@ -0,0 +1,76 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'network'
+  }
+
+NETWORK_OS: junos
+RESOURCE: l3_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+LICENSE: gpl-3.0.txt
+
+DOCUMENTATION: |
+  module: junos_l3_interfaces
+  version_added: 2.9
+  short_description: Manage Layer-3 interface on Juniper JUNOS devices
+  description: This module provides declarative management of a Layer-3 interface on Juniper JUNOS devices
+  author: Daniel Mellado (@dmellado)
+  notes:
+    - 'Tested against JunOS v18.4R1'
+  options:
+    config:
+      description: A dictionary of Layer-3 interface options
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - Full name of interface, e.g. ge-0/0/1
+          type: str
+          required: True
+        unit:
+          description:
+            - Logical interface number. Value of C(unit) should be of type integer
+          default: 0
+        ipv4:
+          description:
+            - IPv4 addresses to be set for the Layer 3 logical interface mentioned in I(name) option.
+              The address format is <ipv4 address>/<mask>, the mask is number in range 0-32
+              for example, 192.0.2.1/24, or C(dhcp) to query DHCP for an IP address
```

Comment:
```suggestion
              for example, 192.0.2.1/24, or C(dhcp) to query DHCP for an IP address.
```

Reply:
Should this description be here or in the IPv4 suboption below?  see https://github.com/ansible-network/resource_module_models/pull/19/files#diff-1d5185730444116efbcb1fbcf059cc30R34 for how eos handles this

Reply:
Well, I think it comes clearer there, but I don't have a strong opinion on that, will update the PR but I can just move there if you think it's better.

Reply:
thanks!